prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -511,6 +511,72 @@ ssize_t rxr_pkt_wait_handshake(struct rxr_ep *ep, fi_addr_t addr, struct rxr_pee
 	return 0;
 }
 
+/*
+ * This function is used for any extra feature that does not have an alternative.
+ *
+ * This function will send a eager rtw packet to trigger handshake.
+ *
+ * We do not send eager rtm packets here because the receiver might require
+ * ordering and an extra eager rtm will interrupt the reorder
+ * process.
+ *
+ * ep: The endpoint on which the packet for triggering handshake will be sent.
+ * peer: The peer from which the sender receives handshake.
+ * addr: The address of the peer.
+ *
+ * This function will return 0 if the eager rtw packet is successfully sent.
+ */
+ssize_t rxr_pkt_trigger_handshake(struct rxr_ep *ep,
+				  fi_addr_t addr, struct rxr_peer *peer)
+{
+	struct rxr_tx_entry *tx_entry;
+	ssize_t err;
+
+	if (peer->flags & RXR_PEER_HANDSHAKE_RECEIVED &&
```

Comment:
should not be "&&" here. 


should be "||".  also can you wrap the two condition each with ()

Reply:
Adopted the `||` symbol. Resolving.